website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -0,0 +1,174 @@
+---
+layout: docs
+page_title: Automated entitlement utilization reporting
+description: >-
+  Learn what data HashiCorp collects to meter Enterprise entitlement utilization. Enable or disable reporting. Review sample payloads and logs.
+---
+
+# Automated entitlement utilization reporting
+
+Automated entitlement utilization reporting sends minimal product-entitlement
+metering data to HashiCorp without requiring you to manually collect and report
+them. It also lets you review your entitlement usage with the monitoring
+solution you already use (Splunk, Datadog, etc.) so you can optimize and manage
+your deployments. 
+
+Utilization reporting shares the minimum data required to validate entitlement
+utilization as contractually obligated. They consist of mostly computed metrics
+and will never contain PII or other sensitive information. Utilization reporting
+shares the data with HashiCorp using a secure, unidirectional HTTPS API, and
+makes an auditable record in the product logs each time it submits a report.
+
+To enable automated reporting you need to upgrade to a version that supports it,
+and open your network to outbound HTTPS traffic on port 443 to
+`https://reporting.hashicorp.services`. If your installation is completely
+air-gapped, you can’t use automated entitlement utilization and can opt out
+instead. You do not need to re-sign the EULA to opt in or out. 
+
+## Enable reporting
+
+To enable automated entitlement utilization reporting, upgrade to a version that
+supports it and make sure that your network allows the right egress traffic. 
+
+### 1. Allow outbound traffic on port 443
+
+Make sure that your network allows HTTPS egress on port 443 to
+https://reporting.hashicorp.services. 
+
+### 2. Upgrade
+
+Upgrade to a release that supports entitlement utilization reporting. These
+releases include:
+
+- [Vault Enterprise 1.14.0](https://releases.hashicorp.com/vault/) and later
+
+
+### 3. Check logs
+
+Automatic entitlement utilization reporting will start sending data within 24
+hours. Check the product logs for records that the data sent successfully. 
+
+Instructions on how to check logs. 
+
+<CodeBlockConfig hideClipboard>
+
+```
+TO DO: Add example log output
+```
+
+</CodeBlockConfig>
+
+If your installation is air-gapped or your network doesn’t allow the correct
+egress, logs will show an error. 
+
+<CodeBlockConfig hideClipboard>
+
+```
+TO DO: Add example error log
+```
+
+</CodeBlockConfig>
```

Comment:
The error I'm currently seeing (with CLS turned off) is as follows:

```
2023-04-14T14:48:17.188-0400 [DEBUG] reporting: beginning snapshot export
2023-04-14T14:48:17.188-0400 [DEBUG] reporting: creating payload
2023-04-14T14:48:17.188-0400 [DEBUG] reporting: marshalling payload to json
2023-04-14T14:48:17.188-0400 [DEBUG] reporting: generating authentication headers
2023-04-14T14:48:17.188-0400 [DEBUG] reporting: creating request
2023-04-14T14:48:17.188-0400 [DEBUG] reporting: sending request
2023-04-14T14:48:17.188-0400 [DEBUG] reporting: performing request: method=POST url=https://census.license.hashicorp.services
2023-04-14T14:48:17.375-0400 [DEBUG] reporting: error status code received: statusCode=403
```

Reply:
Thank you!